packages/core/src/index.ts

```diff
@@ -905,19 +882,17 @@ export class Hopr extends EventEmitter {
       }, 10000)
       try {
         log('Triggering tick channel strategy')
-        await this.tickChannelStrategy()
         await this.updateChannelMetrics()
       } catch (e) {
         log('error in periodic check', e)
       }
       log('Clearing out logging timeout.')
       timer.clear()
-      log(`Setting up timeout for ${this.strategy.tickInterval}ms`)
     }.bind(this)
 
-    log(`Starting periodicCheck interval with ${this.strategy.tickInterval}ms`)
+    log(`Starting periodicCheck interval with 60000ms`)
```

Comment:
This might want to be configurable in HoprdConfig?

Reply:
The periodic check will be gone from TS, as we have the `UniversalTimer` in one of the main loops.
Feel free to remove during your refactor/migration of `Hopr` class.